dsp/modules/gpt3.py

```diff
@@ -173,7 +173,7 @@ def __call__(
                     tokens, logprobs = tokens[:index], logprobs[:index]
 
                 avglog = sum(logprobs) / len(logprobs)
-                scored_completions.append((avglog, self._get_choice_text(c)))
+                scored_completions.append((avglog, self.get_choice_text(c)))
```

Comment:
Hey I'd like to merge this PR, but I think this line is incorrect?

Reply:
Yes, I didn't see that when reverting the refactoring; it's fixed now